src/js/renderers/flash.js

```diff
@@ -237,6 +237,7 @@ const FlashMediaElementRenderer = {
 			}
 			;
 		methods.push('stop');
+		methods.push('canplay');
```

Comment:
Canplay is a native html5 event. Now the swf file trigger it each time when flash open connection to media server. Developers can bind on this method, and trigger play on mediaEl instance